frontend/src/models/api/AcquisitionFile.ts

```diff
@@ -32,5 +33,8 @@ export interface Api_AcquisitionFileProperty extends Api_ConcurrentVersion, Api_
 export interface Api_AcquisitionFilePerson extends Api_ConcurrentVersion, Api_AuditFields {
   id?: number;
   personId?: number;
+  person?: Api_Person;
   personProfileTypeCode?: string;
+  personProfileType?: Api_TypeCode<string>;
+  isDisabled?: boolean;
```

Comment:
Modifying frontend models to match backend model changes. `personId` and `personProfileTypeCode` will need to be removed when backend model + create acquisition team gets refactored